### PR TITLE
Release to production

### PR DESCRIPTION
**Pull request by hiddentao**
* add rollbar and hook up to logrocket

* prettier ignore

* fix lint issues

* fix lint issues

* add config params for ropsten and live, fix bugs

* update env values

* Split time and date

* Abstract dates utility into dates file. Add react timezone

* Add timezone to graphql query

* Detect local timezone

* Set moment to UTC for time

* Style timezone and start date

* Remove timezones import

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by jefflau** [comment]
I just added my timepicker to the release to production

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by jefflau** [comment]
Checked event creation on Rinkeby

**Comment by makoto** [comment]
@jefflau I just created this event using JST timezone for start time as 21:36. However, it still shows as 21:36 in UK. Shouldn't it suppose to say either GMT 6:36 (+8hr) or JST 21:36 ?

https://rinkeby.kickback.events/event/0x19090294ca76461ad2c9bde3e5c773e6d2c41006

**Comment by makoto** [comment]
Another thing I noticed.

It's failing to load event list when it is not logged into Metamask (Chrome and Brave).

Screenshot 2019-03-30 at 21 43 33

When I try this on mainnet site this is not happening.

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by jefflau** [comment]
@makoto why should it show the time in the UK? It's a japanese event and you would want to know the time of the event when you are there, not in the UK

**Comment by jefflau** [comment]
I haven't added the timezone, but I will separately

**Comment by jefflau** [comment]
Have no idea about the events loading. I shouldn't have changed anything that affected that. Any ideas @hiddentao?

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by makoto** [comment]
> Another thing I noticed.
> 
> It's failing to load event list when it is not logged into Metamask (Chrome and Brave).
> 
> Screenshot 2019-03-30 at 21 43 33
> 
> When I try this on mainnet site this is not happening.

I cannot replicate this error any more so I think this was related to some unstable network connection.

**Comment by makoto** [comment]
> I cannot replicate this error any more so I think this was related to some unstable network connection.

Ok, figured out that this is an issue already happening on production. Added as a github issue.

https://github.com/wearekickback/app/issues/125

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by makoto** [comment]
Timezone looks good!

File: src/components/ExternalLinks/EtherScanLink.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const EtherScanLink = ({ address, tx, children }) => (
 
       let link
       if (address) {
-        link = `https://${prefix}etherscan.io/address/${address}`
+        link = `https://${prefix}.etherscan.io/address/${address}`
```
Comment (makoto) [inline src/components/ExternalLinks/EtherScanLink.js]: Are you sure this is correct? The link to etherscan from https://pr121-app-wearekickback.surge.sh/event/0x083b48a2a36cccaa9e34e5e7dbf90aaec450e58f shows `rinkeby..etherscan.io/..`

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr121-app-wearekickback.surge.sh

**Pull request merged**